### DISCUSSION TITLE
biztalk RFID

### DISCUSSION THREAD

**Discussion by mgajownik** [AzureCommunityPL/group#440] [Pomoc]
Cześć jeszcze niedawno było do wykorzystania biztalk server RFID 2013. no ale nie ma
Mozecie pokierowac mnie do instalki tego starego lub zaproponować rozwiazania, które zastępują funkcjonalnie biztalka? Każdy scenariusz jaki przyjdzie Wam do głowy, dla mnie interesujący, ale najbardziej logistyka i obsługa magazynów pod Dynamics365/Dynamics 2012.

M.

**Comment by mgrabarz** [comment]
Niezłe kotlety :)
Szczerze, to bym już kombinował w kierunku IOT Huba i takich zabawek: https://devicecatalog.azure.com/devices?searchTerm=RFID Na Hubie jakaś logika/integracja z Dynamics 365.

**Reply by mgajownik** [reply]
dzieki Marek tak kombinujemy z IoT HUBem
